.github/workflows/local-docker-environment.yml

```diff
@@ -0,0 +1,159 @@
+name: Local Docker Environment
+
+on:
+  push:
+    branches:
+      - trunk
+      - '[6-9].[8-9]'
+      - '[7-9].[0-9]'
+    paths:
+      # Any changes to Docker related files.
+      - '.env.example'
+      - 'docker-compose.yml'
+      # Any changes to local environment related files
+      - 'tools/local-env/**'
+      # These files manage packages used by the local environment.
+      - 'package*.json'
+      # These files configure Composer. Changes could affect the local environment.
+      - 'composer.*'
+      # These files define the versions to test.
+      - '.version-support-*.json'
+      # Changes to this and related workflow files should always be verified.
+      - '.github/workflows/local-docker-environment.yml'
+      - '.github/workflows/reusable-support-json-reader-v1.yml'
+      - '.github/workflows/reusable-test-docker-environment-v1.yml'
+  pull_request:
+    branches:
+      - trunk
+      - '[6-9].[8-9]'
```

Comment:
```suggestion
      - '6.[8-9]'
```

Reply:
I think that the suggested change would require an update once `7.0` is released. As it is currently, it would allow PRs to any WP branch 6.8+ to run.

Reply:
Ah, disregard my previous comment. The view that GH presented did not show the next line for some reason or I missed it.